geth/params/config.go

```diff
@@ -64,6 +66,27 @@ type FirebaseConfig struct {
 	NotificationTriggerURL string
 }
 
+// ReadAuthorizationKeyFile reads and loads FCM authorization key
+func (c *FirebaseConfig) ReadAuthorizationKeyFile() ([]byte, error) {
+	if len(c.AuthorizationKeyFile) <= 0 {
+		return nil, ErrEmptyAuthorizationKeyFile
```

Comment:
In my opinion, it's better to return other error here in order to faster distinguish if the authorization file is set or empty.